src/shogun2-core/src/main/java/de/terrestris/shogun2/service/ModelDescriptionService.java

```diff
@@ -0,0 +1,35 @@
+package de.terrestris.shogun2.service;
+
+import java.io.IOException;
+
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchemaGenerator;
+
+import de.terrestris.shogun2.model.PersistentObject;
+
+/**
+ * Service class to describe classes model.
+ *
+ * terrestris GmbH & Co. KG
+ * @author Kai Volland
+ * @date 04.09.2017
+ *
+ */
+@Service("modelDescriptionService")
+public class ModelDescriptionService<E extends PersistentObject> {
```

Comment:
`<E extends PersistentObject>` is not necessary if no instances of `PersistentObject` are managed by this service.